jiant/utils/utils.py

```diff
@@ -361,15 +401,40 @@ def get_elmo_mixing_weights(text_field_embedder, task=None):
     return params
 
 
-def get_batch_size(batch, keyword="input"):
+def format_output(obj, cuda_devices):
```

Comment:
so this should only receive `torch.Tensor`s? Can you update the function name, docstring, or use a type check to avoid confusion ?

Reply:
It shouldn't only receive torch.Tensor

Reply:
Hrm, I guess I'm still confused as to what this function is doing. My rough understanding is that (1) we're only calling this on the direct outputs of the model (either tensors or lists of tensors?), and if it's the latter (or rather, if we're using Distributed), we want to aggregate that list of tensors into a single tensor. Is that right ? And if so, we can change the name of the function to be more descriptive ?